luma/core/lib.py

```diff
@@ -20,8 +20,9 @@ def __rpi_gpio__(self):
     # a Raspberry Pi... this is imported here so we can swap out the
     # implementation for a mock
     try:
-        import RPi.GPIO
-        return RPi.GPIO
+        import RPi.GPIO as GPIO
+        GPIO.setmode(GPIO.BCM)
```

Comment:
I guess I'm concerned about the amount of times this code can be called.. It could cause a lot of setmode calls whereas these imports are less harmless.